iothub/device/src/InternalClient.cs

```diff
@@ -757,7 +761,13 @@ public Task UploadToBlobAsync(string blobName, Stream source, CancellationToken
                 var context = new PipelineContext();
                 context.Set(_productInfo);
 
-                var transportSettings = new Http1TransportSettings();
+                //Use the proxy of the first transport settings if there is a proxy, regardless of protocol
```

Comment:
space after //

Reply:
Is there a way to make codemaid do that for us?

Reply:
Might be. Take a look at the options dialog.